java/src/jmri/managers/DefaultSignalMastLogicManager.java

```diff
@@ -330,7 +330,11 @@ public int getObjectCount() {
     /** {@inheritDoc} */
     @Override
     public SortedSet<SignalMastLogic> getNamedBeanSet(){
```

Comment:
Could you add a comment mentioning that this is not a live set (i.e. it won't see later changes)?  Having this method here is progress, but eventually it needs to meet the contract of the parent type that this  set is live; that's slowly being built into other parts of the system.

Reply:
Yes, I can add the comment. But it would be good to fix the contract instead.

I got a NPE so I was forced to do something, but I don't fully understand what needs to be done. This was a quick fix to get a working demo.

How do I change this method to meet the contract?